docs/_markdown/markdown-sample.md

```diff
@@ -0,0 +1,153 @@
+An h1 header
+============
+
```

Comment:
We don't need this file right?

Reply:
Nope, it was just used to test the base functionality of Sphinx. I'll remove.